service/src/service.rs

```diff
@@ -129,7 +129,7 @@ pub async fn run() -> anyhow::Result<()> {
     // Load the json-rpc service configuration, which is a combination of the
     // general configuration options for any indexer service and specific
     // options added for JSON-RPC
-    let config = MainConfig::parse(indexer_config::ConfigPrefix::Service, cli.config.as_ref())
```

Comment:
Hey, I think you can `Box::leak` the configuration, that way you don't need to clone things.